auslib/blobs/gmp.py

```diff
@@ -11,6 +11,19 @@ def __init__(self, **kwargs):
         if "schema_version" not in self:
             self["schema_version"] = 1000
 
+    def validate(self, product, whitelistedDomains, checkHash=True):
```

Comment:
I don't think we should support the checkHash=False case. I realize this means you'll need to update some tests to include proper hash lengths, but that's a good thing in the end :).